backend/user-service/package.json

```diff
@@ -19,7 +19,8 @@
     "firebase-admin": "^8.12.1",
     "morgan": "^1.10.0",
     "swagger-ui-express": "^4.1.4",
-    "yamljs": "^0.3.0"
+    "yamljs": "^0.3.0",
+    "node-forge": ">=0.10.0"
```

Comment:
Did you add this manually? AFAIK this won't actually have the intended effect of changing the version of node-forge used by transitive dependencies.

Reply:
orly.. what's the proper way?